Dockerfile

```diff
@@ -5,7 +5,7 @@ COPY . /app
 WORKDIR /app
 
 # Get the go app
-RUN go get -u github.com/banool/codenames-pictures
+RUN go get -u github.com/davidMGS/codenames-pictures
```

Comment:
Does your repo need to be public for this to work?

Reply:
Think for collaborators only